README.md

```diff
@@ -228,6 +229,20 @@ rect rgba(0, 0, 0, 0.1)
 end
 ```
 
+## Discover Initiation Endpoint
+
+PFI's can become publicly discoverable by advertising their IDV endpoint as a [Service](https://www.w3.org/TR/did-core/#services) within their DID Document. In order to increase the likelihood of being discovered, the `service` entry **SHOULD** include the following properties:
+
+| Property          | Value                                                                                                                                                                                            |
+| ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `id`              | MUST be equal to the DID URI with an appended `idv` fragment, example `did:example:123#idv`, therefore the service MAY be [dereferenced](https://www.w3.org/TR/did-core/#did-url-dereferencing). |
```

Comment:
according to the [section on fragments](https://www.w3.org/TR/did-core/#fragment) this would need to be `did:example:123#idv-id` where `idv-id` is the service ID for the IDV service.

we could define our own semantics (per DID method) for a [path query](https://www.w3.org/TR/did-core/#path) for something like `did:example:123/idv` but this seems like it would be a more complicated and unreliable approach.

there's really no good solution unless you know the ID ahead of time. this spec could force a known ID as a workaround.

Reply:
@decentralgabe I couldn't find any language in the DID Core spec nor the RFC 3986 that states that a fragment identifier must be something like `idv-id` and could not be `idv`.  Can you highlight the specific text that led to this conclusion?

The only two examples in the DID Core spec that include a DID service resource  both use an `id` property very similar to the `idv` string proposed in this PR: `agent` and `linked-domain`.

**[Example 5](https://www.w3.org/TR/did-core/#example-a-unique-service-in-a-did-document): A unique service in a DID Document**
```
did:example:123#agent
```

**[EXAMPLE 20](https://www.w3.org/TR/did-core/#example-usage-of-the-service-property): Usage of the service property**
```json
{
  "service": [{
    "id":"did:example:123#linked-domain",
    "type": "LinkedDomains", 
    "serviceEndpoint": "https://bar.example.com"
  }]
}

Reply:
@frankhinek what I said was probably phrased in a confusing manner. My comment is more to say that the value of the fragment must be an _identifier_ instead of a `type` value, so we can guarantee it's uniqueness.